src/stan/lang/generator/expression_visgen.hpp

```diff
@@ -16,22 +17,24 @@ namespace stan {
   namespace lang {
 
     void generate_expression(const expression& e, bool user_facing,
```

Comment:
Is this also a case where it can be included rather than forward declared?  These were here because it didn't use to compile with the includes.

Reply:
it doesn't need to be forward declared or included because `ast.hpp` includes file
file `src/stan/lang/ast/fun/generate_expression.hpp` which has the declaration of `generate_expression`.

cleaning out redundant includes elsewhere.